doc/cephfs/lazyio.rst

```diff
@@ -21,3 +21,57 @@ LazyIO can be enabled by following ways.
 
 - ``ioctl(fd, CEPH_IOC_LAZYIO, 1UL)`` enables LAZY_IO for file handle in
    ceph-fuse mount.
+
+Using LazyIO
+============
+
+ LazyIO includes two methods ``lazyio_propagate()`` and ``lazyio_synchronize()``.
+ With LazyIO enabled, writes may not be visble to other clients until
+ ``lazyio_propagate()`` is called. Reads may come from local cache (irrespective of
+ changes to the file by other clients) until ``lazyio_synchronize()`` is called.
+
+- ``lazyio_propagate(...)`` - Ensures that any buffered writes of the calling client,
+  in the specific region, has been propagated to the shared file.
```

Comment:
explain region. You might as well spell out the names of the arguments to these functions.